### DISCUSSION TITLE
Bring back light theme

### DISCUSSION THREAD

**Discussion by ySergi** [mifi/lossless-cut#1663] [Ideas]
It would be nice to be able to choose between the dark or light theme and not having a default dark theme only. Not everyone uses the PC with the brightness of the screen at maximum in full darkness, so using dark themes in bright environments ends up being annoying for the eyes. So please bring back the clear theme as it was originally in the app.

**Comment by mifi** [comment]
Hi. Not sure what you're asking for, because there is already a light theme, you can toggle it by pressing this button: <img width="33" alt="Screenshot 2023-07-22 at 19 50 31" src="https://github.com/mifi/lossless-cut/assets/402547/0ce48388-7204-40fa-a65c-1224ba94f731">
